src/Redux/missions/missions.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-unused-vars */
+// const REMOVE_MISSION = 'space-travelers-hub/books/REMOVE_MISSION';
```

Comment:
- [ ] It is often considered a good practice to take out unused code instead of leaving them as a comment,  to maintain a clean code based, what do you think?